CHANGES.md

```diff
@@ -2,6 +2,8 @@
 
    * An interface for doing problem tagging in C++ has been added. (#1289)
 
+   * Simplified SDC now only supports C++ reaction networks (#1294)
```

Comment:
Can  you clarify this comment? What we only support now is C++ integrators. You can still use a Fortran network.

Reply:
done